build_tools/cmake/iree_copts.cmake

```diff
@@ -131,6 +131,54 @@ iree_select_compiler_opts(IREE_DEFAULT_COPTS
 set(IREE_DEFAULT_LINKOPTS "${ABSL_DEFAULT_LINKOPTS}")
 set(IREE_TEST_COPTS "${ABSL_TEST_COPTS}")
 
+#-------------------------------------------------------------------------------
+# Sanitizer configurations
+#-------------------------------------------------------------------------------
+
+include(CheckCXXCompilerFlag)
+if(IREE_ENABLE_ASAN)
```

Comment:
wrap variables in in ${} here and below

Reply:
Is it necessary? According to CMake doc

> if(\<constant>)
> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. Named boolean constants are case-insensitive. If the argument is not one of these specific constants, it is treated as a variable or string and the following signature is used.

> if(<variable|string>)
> True if given a variable that is defined to a value that is not a false constant. False otherwise. (Note macro arguments are not variables.)

With `${}` we are going through the first case, but without we are going through the second case?

Reply:
Oh actually I see we are wrapping the variable elsewhere. I'll change it for consistency.